src/devol.cpp

```diff
@@ -235,34 +237,46 @@ void devol(double VTR, double f_weight, double f_cross, int i_bs_flag,
         ta_oldP = ta_newP;                      // have selected NP mutants move on to next generation 
         ta_oldC = ta_newC;
 
-        if (i_check_winner)  {                  // check if the best stayed the same, if necessary 
-            int same = 1;
-            for (int j = 0; j < i_D; j++) {
-                if (t_bestitP[j] != t_bestP[j]) {
-                    same = 0;
-                }
-            }
-            if (same && i_iter > 1)  {
-                i_xav++;
-                memcpy(REAL(par), t_bestP.memptr(), Rf_nrows(par) * sizeof(double));      
-                double tmp_best = ev->eval(par);// if re-evaluation of winner 
-                if (i_av_winner)                //  possibly letting the winner be the average of all past generations 
-                    t_bestC = ((1/(double)i_xav) * t_bestC) + ((1/(double)i_xav) * tmp_best) + 
-                        (d_bestvalit[i_iter-1] * ((double)(i_xav - 2))/(double)i_xav);
-                else
-                    t_bestC = tmp_best;
-            } else {
-                i_xav = 1;
-            }
-        }
-        t_bestitP = t_bestP;
+//         if (i_check_winner)  {                  // check if the best stayed the same, if necessary 
+//             int same = 1;
+//             for (int j = 0; j < i_D; j++) {
+//                 if (t_bestitP[j] != t_bestP[j]) {
+//                     same = 0;
+//                 }
+//             }
+//             if (same && i_iter > 1)  {
+//                 i_xav++;
+//                 memmove(REAL(par), t_bestP.memptr(), Rf_nrows(par) * sizeof(double));      
+//                 double tmp_best = ev->eval(par);// if re-evaluation of winner 
+//                 if (i_av_winner)                //  possibly letting the winner be the average of all past generations 
+//                     t_bestC = ((1/(double)i_xav) * t_bestC) + ((1/(double)i_xav) * tmp_best) + 
+//                         (d_bestvalit[i_iter-1] * ((double)(i_xav - 2))/(double)i_xav);
+//                 else
+//                     t_bestC = tmp_best;
+//             } else {
+//                 i_xav = 1;
+//             }
+//         }
+//         t_bestitP = t_bestP;
+
 
+        //print out temporary results
         if ( (i_trace > 0)  &&  ((i_iter % i_trace) == 0) ) {
             Rprintf("Iteration: %d bestvalit: %f bestmemit:", i_iter, t_bestC);
             for (int j = 0; j < i_D; j++)
                 Rprintf("%12.6f", t_bestP[j]);
             Rprintf("\n");
         }
+        
+        ////////NEW LINE
```

Comment:
That line is redundant.  Git keeps track of this.  You _could_ instead add a comment a la 'changed Jan 2016 by Yi because a, b and c' and actually **explain** why you did it.